tools/ci-scripts/linux/build_cs.zsh

```diff
@@ -5,6 +5,8 @@ set -eux
 nuget update -self
 nuget restore $BOND_ROOT/cs/cs.sln
 
+export PATH=/opt/ghc/bin:$PATH
```

Comment:
This can go in build.zsh around the same place as the `BOOST_ROOT` variable is set. That way, we don't need to repeat it in all the subscripts.

Reply:
done